Jint/Runtime/OrderedSet.cs

```diff
@@ -1,6 +1,6 @@
 namespace Jint.Runtime;
 
-internal sealed class OrderedSet<T>
+public sealed class OrderedSet<T>
```

Comment:
I think this can remain as `internal`, unfortunate need for JS usage where they come up with oddities

Reply:
So `SetConstructor` would handle all constructor requirements and `JsSet` constructor should be internal, like I indicated in earlier discussion, `SetConstructor.Construct()` would have the needed overloads

Reply:
Unfortunately, I wasn't able to compile without this change. I think the alternative would be to downgrade the `JsSet` constructor that uses it to also be `internal`. This could potentially be a breaking change as I don't know where all it is used.

Reply:
I was able to revert this by changing the affected JsSet constructor to internal. Hopefully that doesn't break something elsewhere. I only see it used by SetPrototype.